lib/buffy/throttle_and_timed.ex

```diff
@@ -0,0 +1,363 @@
+# credo:disable-for-this-file Credo.Check.Refactor.LongQuoteBlocks
+defmodule Buffy.ThrottleAndTimed do
+  @moduledoc """
+  This is a variation on the `Buffy.Throttle` behavior.
+
+  It keeps the following functionality:
+    - Wait for a specified amount of time before
+  invoking the work function. If the function is called again before the time has
+  elapsed, it's a no-op.
+
+  Here're what is different:
```

Comment:
We need to s/Here're/Here's but I wonder if we could express this clearer:

```suggestion
  The key difference between `Buffy.Throttle` and `Buffy.ThrottleAndTimed`:
```

Small nitpick: The bullet is capitalized above and all of these are lowercase. Let's pick one

Reply:
Done in b2ce5ed